sites/default/config/core.entity_form_display.node.article.default.yml

```diff
@@ -68,6 +70,18 @@ content:
     third_party_settings: {  }
     type: image_image
     region: content
+  field_paragraph:
+    type: entity_reference_paragraphs
+    weight: 26
+    settings:
+      title: Paragraph
+      title_plural: Paragraphs
+      edit_mode: open
+      add_mode: dropdown
+      form_display_mode: default
+      default_paragraph_type: ''
+    third_party_settings: {  }
+    region: content
   field_tags:
```

Comment:
Does this make sense to the end user? Would someone not familiar with Drupal know what Paragraphs meant?

Reply:
Changed it to page section (the go-to).